### PR TITLE
[3.1] Add instructions on how to recreate remotes #4581

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Add instructions on how to recreate remotes

### CODE REVIEW

File: source/system-administrators/activities/clustering.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :is-up-to-date: True
 
-.. index:: Studio Clustering, Clustering, Auto-Clustering with Studio, Studio's Embedded Database Multi-master Cluster
+.. index:: Studio Clustering, Clustering, Studio's Embedded Database Multi-master Cluster
```
Comment (sumerjabri) [inline source/system-administrators/activities/clustering.rst:3]: Please remove the term "multi-master" from all docs.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: source/system-administrators/activities/clustering/studio-clustering-two-nodes-with-arbiter.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :is-up-to-date: True
 
-.. index:: Setup a Two Node Cluster with Studio and a Studio Arbiter, Auto-Clustering with Studio Example, Studio's Embedded Database Multi-master Cluster Example
+.. index:: Setup a Two Node Cluster with Studio and a Studio Arbiter, Clustering with Studio Example, Studio's Embedded Database Cluster Example
```
Comment (sumerjabri) [inline source/system-administrators/activities/clustering/studio-clustering-two-nodes-with-arbiter.rst:3]: Remove the words Embedded Database, it's just a Studio Cluster. Please do that throughout or ping me if you see a problem.

File: source/system-administrators/activities/kubernetes/studio-clustering-with-kubernetes-deployment.rst [hunk 3]
```diff
@@ -308,7 +308,7 @@ To shutdown the nodes, go to the nodes directory ``kubernetes-deployments/author
       service "authoring-service" deleted
       statefulset.apps "authoring" deleted
 
-Shutting down the nodes one by one allows for a graceful shutdown of the embedded database multi-master cluster.  The ``cascade`` flag allows killing the pods (shutting down the nodes) one by one.  Remember to set the ``cascade`` flag to ``false``, otherwise it will kill both pods at the same time.
+Shutting down the nodes one by one allows for a graceful shutdown of the embedded database cluster.  The ``cascade`` flag allows killing the pods (shutting down the nodes) one by one.  Remember to set the ``cascade`` flag to ``false``, otherwise it will kill both pods at the same time.
```
Comment (sumerjabri) [inline source/system-administrators/activities/kubernetes/studio-clustering-with-kubernetes-deployment.rst:311]: shutdown of the cluster.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: source/system-administrators/activities/clustering/changing-git-url-format-in-cluster.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+============================
+Changing the Cluster Git URL
+============================
+
+When the cluster Git URL for syncing members is changed after a cluster has been setup and started, the nodes on the disk may contain the old URL format when starting up and there may be an error in the logs like below:
+
+   .. code-block:: text
+
+      [ERROR] 2021-03-12T18:54:02,887 [pool-5-thread-10] [job.StudioClockExecutor] | Error executing Studio Clock Job
+      java.lang.ClassCastException: org.eclipse.jgit.transport.TransportHttp cannot be cast to org.eclipse.jgit.transport.SshTransport
```
Comment (avasquez614) [inline source/system-administrators/activities/clustering/changing-git-url-format-in-cluster.rst:16]: This error only occurs when you change the URL from SSH to HTTPS. Maybe we need to clarify that?

**Review by avasquez614 — CHANGES_REQUESTED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**